registry/storage/driver/gcs/gcs.go

```diff
@@ -94,6 +96,10 @@ func FromParameters(parameters map[string]interface{}) (storagedriver.StorageDri
 	}
 
 	var ts oauth2.TokenSource
+	var key struct {
+		ProjectID string `json:"project_id"`
+	}
+	projectID := ""
```

Comment:
this is not necessary. Just reference all calls to `projectID` as `key.ProjectID` instead. No need to retain two variables containing the same data.